tests/functional/syntax/exceptions/test_constancy_exception.py

```diff
@@ -83,37 +83,31 @@ def b():
         """
 interface A:
     def bar() -> uint16: view
+
 @external
 @pure
-def test(to:address):
-    a:A = A(to)
-    x:uint16 = a.bar()
-    """,
-        """
-interface A:
-    def bar() -> uint16: view
-@external
-@pure
-def test(to:address):
-    a:A = A(to)
-    a.bar()
```

Comment:
Deleted this test because it was throwing a StructureException for not assigning the return value to a LHS.

Reply:
i think we should modify these tests instead of deleting them

Reply:
The modification I had in mind was to assign it to a LHS, but that would already be covered by the next test case. What do you think?

Reply:
idts -- the next one is an extcall, not a staticcall

Reply:
ah, the previous one -- yep, it's a dup!